test/kitchen_sink/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ target_compile_options(kitchen_sink_options INTERFACE
         -Wfloat-equal
         -Wmissing-format-attribute
         -Wno-long-long
+	$<$<COMPILE_LANGUAGE:C>:-Wstrict-prototypes>
```

Comment:
can you please fix the indent (8 spaces)

Reply:
Done.  Also re-based against origin/develop and fixed new instance of the warning in unqiue_iq.c

Pushed revised branch.